pkg/queue/concurrency_state_test.go

```diff
@@ -189,21 +236,31 @@ func TestConcurrencyStateResumeRequest(t *testing.T) {
 		}
 	}))
 
-	resume := Resume(ts.URL)
-	if err := resume(); err != nil {
+	logger := ltesting.TestLogger(t)
+	tempDir := t.TempDir()
+	tokenFile := createTempTokenFile(logger, tempDir)
```

Comment:
I think we can use `os.WriteFile` here and drop `createTempTokenFile`.

```suggestion
	tokenPath := filepath.Join(t.TempDir(), "secret")
	if err := os.WriteFile(tokenPath, []byte("0123456789"), 0700); err != nil {
		t.Fatal(err)
	}	
```